DatadogRUM/Sources/Integrations/CrashReportReceiver.swift

```diff
@@ -359,14 +363,17 @@ internal struct CrashReportReceiver: FeatureMessageReceiver {
                 type: errorType
             ),
             os: lastRUMView.os,
+            parentView: nil,
             service: lastRUMView.service,
             session: .init(
                 hasReplay: lastRUMView.session.hasReplay,
                 id: lastRUMView.session.id,
-                type: lastRUMView.ciTest != nil ? .ciTest : .user
+                type: lastRUMView.ciTest != nil ? .ciTest : (lastRUMView.synthetics != nil ? .synthetics : .user)
             ),
             source: lastRUMView.source?.toErrorEventSource ?? .ios,
-            synthetics: nil,
+            synthetics: lastRUMView.synthetics.map({ st in
+                    .init(injected: false, resultId: st.resultId, testId: st.testId)
+            }),
```

Comment:
```suggestion
            synthetics: lastRUMView.synthetics.map {
                    .init(injected: false, resultId: $0.resultId, testId: $0.testId)
            },
```

again - just a showcase :)